driver-sync/src/main/com/mongodb/client/Clock.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client;
+
+/**
+ * A clock to wrap time operations.
+ *
+ * @since 3.11
+ */
+public interface Clock {
+    /**
```

Comment:
Once we remove Java 6 and 7 support, we could remove this interface and use the Java 8 classes `java.time.Clock` and `java.time.Instant` instead.

Reply:
Acknowledged